README.md

```diff
@@ -1,13 +1,12 @@
-# Troubleshooting as a serivce
+# Troubleshooting as a service using Must-Gather
```

Comment:
Do you see a benefit in only committing to Must-Gathers?
I mean, right now we don't have any other reliable way, but in the general case, we could make it work with cluster dump or even connecting to a live cluster. I think plain Kubernetes doesn't operate on Must-Gathers.

Reply:
@vladikr I updated based on your comment. Please review.